src/azul/plugins/metadata/anvil/service/response.py

```diff
@@ -89,8 +89,22 @@ class AnvilSearchResponseStage(SearchResponseStage):
 
     def process_response(self, response: ResponseTriple) -> MutableJSON:
         hits, pagination, aggs = response
+        hits = list(map(self._make_hit, hits))
+        self.inject_accessible(hits)
+        if self.entity_type == 'files':
+            self.inject_file_urls(hits, ('files',))
```

Comment:
When I suggested changing the type annotation of the `path` parameter from `list` to `Sequence`, it was not my intent to suggest changing the types of any of the provided arguments. List literals are preferable here because they are easier to read, being less ambiguous and (in this case) shorter.

```suggestion
            self.inject_file_urls(hits, ['files'])
```